source/Missions/Services/Agents/Messages/ConfirmedNetworkAgentDamaged.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Common.Messaging;
+using Common.Serialization;
+using GameInterface.Serialization;
+using GameInterface.Serialization.External;
+using ProtoBuf;
+using System;
+using TaleWorlds.MountAndBlade;
+
+namespace Missions.Services.Agents.Messages
+{
+    /// <summary>
+    /// External event for agent damage
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public readonly struct ConfirmedNetworkAgentDamaged : INetworkEvent
```

Comment:
With this, we would have two NetworkAgentDamaged that has to do different things, I could add a bool to it but it would complicate understanding in my opinion.

Reply:
I see that you wanted to change the other one now, so nevermind